File: 2 período/aeds2/prova1/q2174.c
#include <stdio.h>
#include <string.h>


int main() {
  int n;
  scanf("%d", &n);

  char pokemos[n][1000];
  int cont = 0;
  
  while (n--) {
    char pok[1000];
    scanf("%s", pok);
    int present = 0;

    for (int i = 0;i < cont; i ++) {
      if (!strcmp(pok, pokemos[i])) {
        present = 1;
        break;
      }
    }

    if (!present) {
      strcpy(pokemos[cont], pok);
      cont++;
    }

  }

  int res = 151 - cont;
  printf("Falta(m) ");
  printf("%d", res);
  printf(" pomekon(s).\n");
  return 0;
}

File: 2 período/aeds2/prova1/q2174.java
import java.util.*;

public class q2174 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    int n = sc.nextInt();
    sc.nextLine();
    int cont = 0;

    String[] pokemons = new String[n];
    for (int i =0; i < n; i ++) {
      boolean present = false;
      String pok = sc.nextLine();
      for (int j = 0; j < cont; j ++) {

        if (pokemons[j].equals(pok)) {
          present = true;
          break;
        }
      }

      if (!present) {
        pokemons[cont] = pok;
        cont++;
      }
    }

    int res = 151 - cont;
    System.out.println("Falta(m) " + res + " pomekon(s).");
    sc.close();
  }
}
